.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2.1
+executors:
+  node-11:
+    docker:
+      - image: circleci/node:10
+jobs:
+  build:
+    executor: node-11
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            # when lock file changes, we still use increasingly general patterns to restore cache
+            # the install afterwards will only install packages with different versions
+            - yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - yarn-cache-{{ .Branch }}-
+      - run:
+          name: Install dependencies
+          command: |
+            yarn --version
+            yarn install --frozen-lockfile
+            cd example
+            yarn install --frozen-lockfile
+      - save_cache:
+          key: yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.yarn
+            - ~/.cache/yarn
+            - ./node_modules
+            - ./example/node_modules
+      - run:
+          name: Test Library
+          command: |
+            yarn test:ci
+      - run:
+          name: Test Example App
```

Comment:
Yeah, tried first put it in one step, but I preferred the immediate overview in the circleci-panel, in which part of the app the tests would fail. See [this example](https://circleci.com/gh/wtho/jest-preset-angular/10)

If you prefer them to be summarized, I can reduce them to one step.

Reply:
Oh now I got you, overlooked **workflows**
Will set that up :smile:

Reply:
Ok, so I made it work.
The overview is more extendable and tests run parallel, but currentlly we lose about 30s for not running in one workspace (workspace has to be persisted and attached between different jobs).

I still prefer it, as it gives a better overview, especially if we might later choose to add linting, generating docs, ... (see [workflow example](https://circleci.com/workflow-run/f90a4a1e-1317-42d3-9be8-38b8967d7feb) from react-native-paper)